Modelica/ComplexBlocks/ComplexMath/TransferFunction.mo

```diff
@@ -15,9 +15,16 @@ protected
   Complex aw[size(a,1)];
   Complex bSum;
   Complex aSum;
+  function PowerOfI "Internal function to compute j^k for any integer"
```

Comment:
Function names should start with a lower-case letter.

Should it be an encapsulated function?

Reply:
I will change first letter. I don't think it should be encapsulated, but perhaps moved to another place in the future.